### DISCUSSION TITLE
ruby is loading libcrypto in an unsafe way

### DISCUSSION THREAD

**Discussion by sparsh9** [openssl/openssl#21419] [Q&A]
```
WARNING: /usr/local/opt/ruby@2.7/bin/ruby is loading libcrypto in an unsafe way
zsh: abort      rails db:create
```

Hi Everyone, 
I am trying to run rails in Macbook Pro (2017 Intel Chip), but I have been running into this issue. 
I have tried many solutions but no result. 
Please help me with this.

**Comment by t8m** [comment]
I am sorry but this rather looks like question for Ruby developers. We do not have any experience with using OpenSSL with Ruby on Macs

**Reply by sparsh9** [reply]
Could you recommend any forum where I can post this error?
@t8m

**Reply by paulidale** [reply]
Try: https://gprivate.com/65rsz
